xml/System.Text.Json/JsonSerializer.xml

```diff
@@ -306,8 +306,11 @@ For more information, see [How to serialize and deserialize JSON](/dotnet/standa
 
 ]]></format>
         </remarks>
+        <exception cref="T:System.IndexOutOfRangeException">
+          .NET Core 3.0 and earlier versions only: <typeparamref name="TValue" /> is <see cref="T:System.Char" /> and <paramref name="json" /> is an empty string.</exception>
```

Comment:
i believe System.Text.Json is introduced in 3.0, no earlier versions
```suggestion
          .NET Core 3.0 only: <typeparamref name="TValue" /> is <see cref="T:System.Char" /> and <paramref name="json" /> is an empty string.</exception>
```

Reply:
True, it is built-in starting .NET Core 3.0 and above, but the user can grab the NuGet package which targets netstandard2.0, so you can run it on .NET Core 2.0, 2.1, and .NET 4.7.2 (or above).
https://www.nuget.org/packages/System.Text.Json/5.0.0-rc.1.20451.14

Reply:
Thanks, I'll update it.

Reply:
That said, to clarify, if the user grabs the latest version of the NuGet package, they will get the correct/new behavior. So, what you have is good as-is.

If someone uses the older NuGet package, say [4.6.0](https://www.nuget.org/packages/System.Text.Json/4.6.0), that will have the IOOR behavior. That version shipped at the same time as the .NET Core 3.0 release.

Anything newer than that will have the right behavior.